dotenv.cabal

```diff
@@ -66,9 +66,9 @@ executable dotenv
                        , base-compat >= 0.4
                        , dotenv
                        , optparse-applicative >= 0.11 && < 0.19
-                       , megaparsec
-                       , process
-                       , text
+                       , megaparsec >= 9.2.2 && <= 9.4.1
+                       , process >= 1.6.16 && < 1.7
+                       , text >= 1.2.5 && < 1.3
```

Comment:
@DanRCM I  see we're also using this dependencies in the library. It's not necessary to define the boundaries for every dependency in library, executable, and test-suite. It's good practice to define them in one section it once and the boundaries will automatically be inherited by other sections.